src/Symfony/Component/Form/Resources/translations/validators.eu.xlf

```diff
@@ -14,6 +14,126 @@
                 <source>The CSRF token is invalid.</source>
                 <target>CSRF tokena ez da egokia.</target>
             </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid HTML5 color.</source>
+                <target>Balio hori ez da HTML5 kolore onargarria.</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>Mesedez, sartu baliozko urtebetetze-eguna.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>Hautatutako aukera ez da egokia.</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>Bilduma ez da baliozkoa.</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>Mesedez, hautatu baliozko kolore bat.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>Mesedez, hautatu baliozko herrialde bat.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>Mesedez, hautatu baliozko txanpon bat.</target>
```

Comment:
Could you double check this? Does it say "coin" or "currency"?

Reply:
You can use "moneta" or "txanpon". I updated the translation.